app/models/salt_event.rb

```diff
@@ -10,6 +10,11 @@ class SaltEvent < ApplicationRecord
   PROCESS_TIMEOUT_MIN = 5
   NO_EVENTS_TIMEOUT_SEC = 5
 
+  ENABLED_EVENT_HANDLERS = [
+    SaltHandler::MinionStart,
+    SaltHandler::MinionHighstate
+  ].freeze
```

Comment:
I don't think I can do that here. There is no database column involved. This is a list of classes which we ask wether they are capable of handling a particular event or not. They should all implement the ```can_handle_event?``` method.